STABLE_RELEASES.md

```diff
@@ -0,0 +1,88 @@
+priority is to minimise the risk caused by changes not strictly required to fix qualifying bugs; this tends to
+be correlated with minimising the size of such changes. As such, the same bug may need to be fixed in different
+ways in stable releases and `master` branch.
+
+### What qualifies as SRU
+
+* **High-impact bugs**
+  * Bugs that may directly cause a security vulnerability.
+  * *Severe regressions* from a Cosmos-SDK's previous release. This includes all sort of issues
+    that may cause the core packages or the `x/` modules unusable.
+  * Bugs that may cause **loss of user's data**.
+* Other safe cases:
+  * Bugs which don't fit in the aforementoned categories for which an obvious safe patch is known.
+  * Relatively small yet strictly non-breaking changes that introduce forward-compatible client
+    features to smoothen the migration to successive releases.
+
+### What does not qualify as SRU
+
+* State machine changes.
+* New features that introduces API breakages (e.g. public functions removal/renaming).
+* Cosmetic fixes, such as formatting or linter warning fixes.
+* Documentation fixes.
+
+## Code Owner Membership
+
+In the ethos of open source projects, and out of necessity to keep the code
+alive, the core contributor team will strive to permit special repo privileges
+to developers who show an aptitude towards developing with this code base.
+
+Several different kinds of privileges may be granted however most common
+privileges to be granted are merge rights to either part of, or the entire the
+code base (though the github `CODEOWNERS` file). The on-boarding process for
+new code owners is as follows: On a bi-monthly basis (or more frequently if
+agreeable) all the existing code owners will privately convene to discuss
+potential new candidates as well as the potential for existing code-owners to
+exit or "pass on the torch". This private meeting is to be a held as a
+phone/video meeting. Subsequently at the end of the meeting, one of the existing
+code owners should open a PR modifying the `CODEOWNERS` file. The other code
+owners should then all approve this PR to publicly display their support.
+
+Only if unanimous consensus is reached among all the existing code-owners will
+an invitation be extended to a new potential-member. Likewise, when an existing
+member is suggested to be removed/or have their privileges reduced, the member
+in question must agree on the decision for their removal or else no action
+should be taken. If however, a code-owner is verifiably shown to intentionally
+have had acted maliciously or grossly negligent, code-owner privileges may be
+stripped with no prior warning or consent from the member in question.
+
+Earning this privilege should be considered to be no small feat and is by no
+means guaranteed by any quantifiable metric. It is a symbol of great trust of
+the community of this project.
```

Comment:
Shouldn't this process reflect that outlined in the [Cosmos GitHub Orgs Document](https://docs.google.com/document/d/1QV5Hrzq3jVTnn40QyvN6hxZkoxXRd8-n4GbHBI6_tJw/edit#)?

Reply:
aha I see this is with regard specifically to stable releases